app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.kt

```diff
@@ -139,11 +130,9 @@ class DescriptionEditFragment : Fragment() {
 
         binding.fragmentDescriptionEditView.setLoginCallback {
             val loginIntent = LoginActivity.newIntent(requireActivity(),
-                    LoginActivity.SOURCE_EDIT, funnel.sessionToken)
+                    LoginActivity.SOURCE_EDIT, UUID.randomUUID().toString())
```

Comment:
It looks like the `EDIT_SESSION_TOKEN` in the `LoginActivity` is not actually used. Would it be possible to delete it from `LoginActivity.newIntent()`?

Reply:
Done